app/models/sub_section.rb

```diff
@@ -0,0 +1,67 @@
+class SubSection
+  attr_reader :sub_section_slug
+
+  delegate :title, to: :tag_from_content_api
+
+  def initialize(sub_section_slug)
+    @sub_section_slug = sub_section_slug
+  end
+
+  def lists
+    if curated_links?
+      @lists ||= curated_lists
+    else
+      @lists ||= fallback_list_of_uncurated_links
+    end
+  end
+
+  def curated_links?
+    subsection_from_content_store.details && subsection_from_content_store.details.groups
+  end
+
+  def exists?
+    tag_from_content_api.present?
+  end
+
+private
+
+  def curated_lists
+    subsection_from_content_store.details.groups.map do |group|
+      links = group.contents.map do |url|
+        tagged_items_from_content_api.find { |artefact| artefact.id == url }
+      end
```

Comment:
I think we need a `.compact` call here to remove any nil items.  This could happen if the group contains any items that are no longer tagged to this subtopic.